test/test_transforms_tensor.py

```diff
@@ -579,7 +620,8 @@ def test_convert_image_dtype_save_load(tmpdir):
 
 
 @pytest.mark.parametrize("device", cpu_and_gpu())
-@pytest.mark.parametrize("policy", [policy for policy in T.AutoAugmentPolicy])
+# TODO: Why are there failures only for CIFAR10??
```

Comment:
I'll check that

Reply:
(it might have been fixed since then, I'm not sure)

Reply:
No, it is failing here and in proto tests